common/utils.go

```diff
@@ -137,3 +139,20 @@ func GetBridgeNetDev(bridgeName string) ([]NetDev, error) {
 		return link.Attrs().Name == bridgeName
 	})
 }
+
+// Do post-attach configuration of all veths we have created
+func ConfigureARPforVeths(processID int, prefix string) error {
+	_, err := FindNetDevs(processID, func(link netlink.Link) bool {
+		ifName := link.Attrs().Name
+		if strings.HasPrefix(ifName, prefix) {
```

Comment:
I was thinking we only want plugin-generated interfaces rather than all weave interfaces.  In context that probably doesn't matter. 
The `GetWeaveNetDevs` code would need a bit of refactoring to be re-used here.